plugins/application/loki/pkg/lib/loki.go

```diff
@@ -0,0 +1,51 @@
+package lib
+
+import (
+	"time"
+	"fmt"
+
+	"github.com/infrawatch/apputils/connector"
+	"github.com/infrawatch/sg-core/pkg/data"
+)
+
+type LokiConfig struct {
+	Connection  string `validate:"required"`
+	BatchSize   int
+	MaxWaitTime int
+}
+
+// Creates labels used by Loki. Discards everything, that isn't
+// a string
+func createLabels(rawLabels map[string]interface{}) (map[string]string, error) {
+	result := make(map[string]string)
+	for key, value := range rawLabels {
+		if typedValue, ok := value.(string); ok {
+			result[key] = typedValue
+		}
+	}
+	if len(result) == 0 {
+		return nil, fmt.Errorf("Unable to create log labels")
```

Comment:
Error message shouldn't be capatalized

Reply:
Didn't know this.

Reply:
I actually found a few more capitalized error messages